### DISCUSSION TITLE
Subject: Request to unlink student email address from GitHub account

### DISCUSSION THREAD

**Discussion by doancongbachcdit** [community/community#119916] [GitHub Education]
### Hi everyone,

__Dear GitHub Support Team,

I am [Doan Cong Bach], a GitHub user, and I would like to request your assistance in resolving an issue related to registering for the student discount on my GitHub account.

I used my student email address, [dcbach.dhti15a1nd@sv.uneti.edu.vn], to register for the student discount on my old GitHub account ([doancongbach]). However, for some reasons, I had to cancel the registration of my old GitHub account and create a new one ([doancongbachcdit]).

When I attempted to register for the student discount again on my new account, I received a notification that the student email address had been used for registration previously. Therefore, I would like to request the GitHub Support Team to unlink the student email address ([dcbach.dhti15a1nd@sv.uneti.edu.vn]) from my old GitHub account ([doancongbach]) so that I can use it to register again on the new account.

Please confirm my request and notify me when this process has been completed. Thank you for taking the time to assist me.

Best regards,
[Doan Cong Bach]
[doancongbachcdit]

**Comment by samus-aran** [comment]
Hi @doancongbachcdit  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
